### ISSUE TITLE
GPL-nnn Update the 

### ISSUE THREAD

**Issue by Chris-Friend** [sanger/General-Backlog-Items#93]
**User story**
In adopting the Beckman workflow, event reporting is changing. This will break anything dependent on these events. One such thing is the filtering for cherrypicked samples in both lighthouse and crawler. These are SQL queries which currently query for the Sentinel cherrypicking events only. They need to be updated to query for Beckman picking events **as well**.

The alternative is to fire the same `cherrypick_layout_set` event for the Beckman process, but since no real Beckman action corresponds to this event, this solution has been rejected.

**Who are the primary contacts for this story**
Katy T
Phil
Chris F

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Update lighthouse method to get cherrypicked samples to query for both Sentinel or Beckman cherrypicking events. This is used to generate the positives report.
- [ ] Update crawler method to get cherrypicked samples to query for both Sentinel or Beckman cherrypicking events (introduced as part of https://github.com/sanger/crawler/issues/144)
- [ ] (**Optional/Do as a separate task**) - have only a single implementation of these "is a sample cherrypicked" rules

**Dependencies**
This story is blocked by the following dependencies:
- https://github.com/sanger/crawler/issues/144

**Additional context**

**Comment by Chris-Friend** [comment]
Joins on sample UUID. Looks for the Beckman destination plate creation event type for a specific sample
```
SELECT mlwh_sample.description AS `root_sample_id`, mlwh_lh_sample.plate_barcode AS `plate_barcode`, 
mlwh_sample.phenotype AS `Result_lower`, mlwh_lh_sample.coordinate AS `coordinate`
FROM mlwhd_mlwarehouse_devdata.sample AS mlwh_sample
JOIN mlwhd_mlwarehouse_devdata.lighthouse_sample AS mlwh_lh_sample ON (mlwh_sample.uuid_sample_lims = mlwh_lh_sample.lh_sample_uuid)
JOIN mlwhd_mlwh_events_proddata.subjects AS mlwh_events_subjects ON (mlwh_events_subjects.uuid = UNHEX(REPLACE(mlwh_lh_sample.lh_sample_uuid, "-", "")))
JOIN mlwhd_mlwh_events_proddata.roles AS mlwh_events_roles ON (mlwh_events_roles.subject_id = mlwh_events_subjects.id)
JOIN mlwhd_mlwh_events_proddata.events AS mlwh_events_events ON (mlwh_events_events.id = mlwh_events_roles.event_id)
JOIN mlwhd_mlwh_events_proddata.event_types AS mlwh_events_event_types ON (mlwh_events_event_types.id = mlwh_events_events.event_type_id)
-- f" WHERE mlwh_sample.description IN %(root_sample_ids)s"
-- f" AND mlwh_lh_sample.plate_barcode IN %(plate_barcodes)s"
AND mlwh_events_event_types.key = "lh_beckman_cp_destination_created"
GROUP BY mlwh_sample.description, mlwh_lh_sample.plate_barcode, mlwh_sample.phenotype, mlwh_lh_sample.coordinate;
```
